lib/galaxy/webapps/galaxy/fast_app.py

```diff
@@ -154,7 +154,7 @@ def _check_legacy_openapi_does_not_increase(legacy_openapi):
     After migrating any legacy route to FastAPI, remember to update `expected_legacy_paths` with the current number
     until all routes are migrated, then, completely remove this check function.
     """
-    expected_legacy_paths = 169
+    expected_legacy_paths = 170
```

Comment:
Can you log the number of found and the number of expected ?

Reply:
that is it, 169 for Galaxy tests, 170 for TS